### PR TITLE
Mention standard/pro account for IVPN

### PR DESCRIPTION
**Pull request by dngray**
https://deploy-preview-1571--privacytools-io.netlify.com/providers/vpn/

### CODE REVIEW

File: _includes/sections/vpn.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h5><span class="badge badge-info">Extra Functionality</span></h5>
       <img src="/assets/img/svg/3rd-party/ivpn.svg" height="70" width="200" class="img-fluid d-block mr-auto ml-auto align-middle" alt="IVPN">
     </div>
     <div class="col">
-    <h2>IVPN <span class="badge badge-info">USD $100/Year</span></h2>
+    <h2>IVPN <span class="badge badge-info">USD $60/Year Standard</span> or <span class="badge badge-info">USD $100/Year Pro</span></h2>
```
Comment (JonahAragon) [inline _includes/sections/vpn.html]:
```suggestion
    <h2>IVPN <span class="badge badge-info">Standard USD $60/Year</span> <span class="badge badge-info">Pro USD $100/Year</span></h2>
```

see: L57

Comment (dngray) [reply _includes/sections/vpn.html]: Yeah true, I do like that better.

File: _includes/sections/vpn.html [hunk 2]
```diff
@@ -79,7 +79,7 @@ <h5><span class="badge badge-info">Extra Functionality</span></h5>
       <img src="/assets/img/svg/3rd-party/ivpn.svg" height="70" width="200" class="img-fluid d-block mr-auto ml-auto align-middle" alt="IVPN">
     </div>
     <div class="col">
-    <h2>IVPN <span class="badge badge-info">USD $100/Year</span></h2>
+    <h2>IVPN <span class="badge badge-info">Standard USD $60/Year</span> <span class="badge badge-info">Pro USD $100/Year</span></h2>
```
Comment (dawidpotocki) [inline _includes/sections/vpn.html]:
```suggestion
    <h2>
      IVPN
      <span class="badge badge-info">Standard USD $60/Year</span>
      <span class="badge badge-secondary">Pro USD $100/Year</span>
    </h2>
```

Comment (dngray) [reply _includes/sections/vpn.html]: Yes, right as always.

Comment (JonahAragon) [reply _includes/sections/vpn.html]: Should ProtonVPN be updated to match then?

**Comment by djoate** [comment]
Maybe a similar PR can be done for ProtonVPN and their plans

**Pull request merged**